LICENSE.md

```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2018 Digital Bazaar, Inc.
```

Comment:
there was mention of maybe updating all copyrights, please don't forget this one.